trace.go

```diff
@@ -49,6 +49,10 @@ const (
 	TraceBatchReplication
 	// TraceBatchKeyRotation will trace batch keyrotation operations.
 	TraceBatchKeyRotation
+	// TraceBatchExpire will trace batch expiration operations.
```

Comment:
It is better to have newer flags appear after all the current flags. This way the enum value for these constants remain the same across madmin-go versions.

Reply:
the reason is it follow a more easier-to-read convention like `TraceBatch` so that we can avoid disjointed constants.

Also enum is compiled in so there is no problem as such.